tests/vxlan/test_vxlan_bfd_tsa.py

```diff
@@ -610,13 +626,12 @@ def test_tsa_case6(self, setUp, encap_type):
 
         self.verfiy_bfd_down(ep_list)
 
-        duthost.shell("sudo config reload -y",
-                      executable="/bin/bash", module_ignore_errors=True)
-        assert wait_until(300, 20, 0, config_system_checks_passed, duthost, [])
+        config_reload(duthost, safe_reload=True, check_intf_up_ports=True)
```

Comment:
Same as above

Reply:
@bingwang-ms 'wait_for_bgp=True' option in config_reload??

Reply:
Yes, exactly